### ISSUE TITLE
[BLU] Can't do the level 80 opener in time

### ISSUE THREAD

**Issue by Wild-Carrot** [Nik-Potokar/XIVSlothCombo#1324]
**Class/Job:** Blue Mage
**Feature:** New Moonflute Opener (Level 80 Edition)

**Issue:**  I cannot do the full opener. I have a 2.23 GCD and this opener should be doable on a 2.5, but I run out of time either right as I hit Matra Magic, or right before. In both cases, I lose Phantom Flurry. I had no issues with the older level 70 opener. This could be an issue on my end, but this isn't something I can verify by myself, or if it is, I don't know how. I've considered that it might be a ping issue, but for starters, my ping isn't horrendous, and also, I use xivalexander anyway.

And for the record, this is referring to the normal level 80 opener, not the DoT variant.

Also, J Kick isn't in the Moon Flute window with this plugin. It uses it right before Moon Flute.

**Edit:** I started to look at what the plugin is doing and I think I see the issue. I didn't spot it at first.

It happens due to the first Winged Reprobation. Between Wing 1 and Wing 2, it's supposed to weave in two oGCDs (Feather and Sea Shanty). However, it's not weaving in that window but instead delaying it until after Wing 2, at which point it dumps Feather, Shanty, Shock Strike, and Being Mortal, which causes some time loss.

It looks like it's trying to weave Feather, and I'd assume Shanty too, after that first Wing, as the icon changes to Feather, but before it can get it off, the icon swaps back to Wing for Wing 2.

If this is something on my end, just let me know.

**Comment by Taurenkey** [comment]
I've checked the winged opener and it seems to work as intended. 
![image](https://github.com/Nik-Potokar/XIVSlothCombo/assets/30053704/111d1c2b-8d39-454f-8b8b-5e4ecf927ae7)

As it is based off [this](https://lensdump.com/i/qpdS4D), which as you can see does manage to follow the order correctly.

**Comment by Wild-Carrot** [comment]
Hmm, I wonder why your first Wing Reprobation is allowing two weaves, but mine is not. Manually doing it right is easy enough, especially at my SpS, but yeah, my scombo holds the oGCDs after the first Wing, then dump both plus the other two after the second wing, as mentioned.

Because of my SpS, I sometimes get it all off in time, but it's not doing it in the same order as you.

I'm guessing this is just an xivalexander confliction, as that is the only other combat related mod/plugin I use. It's clearly on my end, but I can't think of anything else that would even be related to this. Seems like a dead end, but thank you for checking that it was working well on your end.

**Comment by FFXIVWytes** [comment]
I found my culprit anyway. I didn't have ground action queue enabled in the mouseover plugin I was using so it wasn't casting feather rain which threw off the combo. Solved for me anyway!

**Comment by Wild-Carrot** [comment]
Alright, after some trial and error messing around, I got it working with the help of other plugins. Going to post it incase anyone is curious.

With just scombo, I could not get it to work, but we already knew that it does considering someone showed it working by itself. I turned off xivalexander's latency and timing helper, but to no avail. I also tried disabling "Press Action twice to execute" in the game settings, but that didn't work, either.

So then I went back to scombo + ReAction (plugin for automatically setting spells to cast on specified target when pressed, like getting Feather Rain to always land in the middle of the target). It's important that I mention Feather Rain here because it's actually the culprit of my problems. I originally disabled the mouseover plugin to verify that it wasn't causing the issue, but now that I'm just looking for a solution, it was time to enable it.

Originally, same hang up. oGCDs get to Feather Rain, then wait a GCD after it's supposed to fire off where scombo then makes me drop 4 oGCDs in a row, causing time loss. But ReAction has a specific option that instantly fires off AoEs where your mouse is. It worked. Not the solution I want, but it is a solution.

So then I tried Mouseover Action. Used it in the past and seems like a more limited version of ReAction. Added Feather Storm to it, and set it to use it on my target... and just like that, the entire BLU opener worked perfectly fine.

For whatever reason, my scombo 'breaks' because of Feather Rain itself. It seems like an issue specific to me, which is fortunate as I was only using this for testing purposes, but maybe if someone else ends up in this situation, it can give them a solution or at the very least, a place to start.
